addons/account_followup/account_followup.py

```diff
@@ -235,18 +199,18 @@ def get_followup_table_html(self, cr, uid, ids, context=None):
         """
         from report import account_followup_print
 
-        assert len(ids) == 1
-        if context is None:
-            context = {}
-        partner = self.browse(cr, uid, ids[0], context=context)
+        assert len(self.ids) == 1
+        partner = self.browse(self.id)
         #copy the context to not change global context. Overwrite it because _() looks for the lang in local variable 'context'.
         #Set the language to use = the partner language
-        context = dict(context, lang=partner.lang)
+        ctx = self._context.copy()
+        ctx['lang'] = partner.lang
+        self.with_context(ctx)
```

Comment:
Above statement does nothing, to verify /cc @rco-odoo .

with_context() does not change the context of a given recordset(s), instead it returns an equivalent recordset(s) with the modified context.

So, if you want to invoke a method with a specific context, make a recordset(s) with that context and invoke the method on that one.